node/src/accountant/payable_dao.rs

```diff
@@ -1077,18 +1079,21 @@ mod tests {
 
     #[test]
     fn range_query_does_not_display_values_from_below_1_gwei() {
-        let timestamp = now_time_t() - 5000;
-        let main_setup = |insert: &dyn Fn(&str, i128, i64, Option<i64>)| {
+        let timestamp_1 = now_time_t() - 11_001;
+        let timestamp_2 = now_time_t() - 5000;
```

Comment:
I notice you're calling now_time_t() twice. It'd be better to call it once and use the result twice.